codeSnippets/snippets/embedded-server/src/main/kotlin/com/example/Application.kt

```diff
@@ -53,3 +53,20 @@ fun runConfiguredServer() {
         }
     }.start(wait = true)
 }
+
+fun runServerWithCommandLineConfig(args: Array<String>) {
+    embeddedServer(
+        factory = Netty,
+        configure = {
```

Comment:
There should be an overload that accepts the `ServerConfig`

Reply:
You mean using the `rootConfig` param? Can you suggest what it would look like? Because i couldn't get it to work